src/test/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServerSpec.scala

```diff
@@ -215,7 +215,7 @@ class JsonRpcHttpServerSpec extends AnyFlatSpec with Matchers with ScalatestRout
       status shouldEqual StatusCodes.TooManyRequests
     }
 
-    fakeClock.advanceTime(2 * serverConfigWithRateLimit.rateLimit.minRequestInterval.toMillis)
+    Thread.sleep(30)
```

Comment:
I'm not sure how this works, but it looks better to have a mock clock that we can move as we want rather than sleeping threads.

Reply:
This is possibly the weakest point of all the PR, but I would remove that test anyway, because the ability to drop the cache is covered in another test (`SimpleLRUSpec`). So it might be an overkill to test it again and provide an external clock to test it.

Reply:
Done